engines/python/setup/djl_python/transformers-neuronx.py

```diff
@@ -229,18 +229,18 @@ def infer(self, inputs):
 
             if self.enable_streaming:
                 outputs.add_property("content-type", "application/jsonlines")
-                if self.enable_streaming == "huggingface":
-                    outputs.add_stream_content(
-                        StreamingUtils.use_hf_default_streamer(
-                            self.model, self.tokenizer, input_text, None,
-                            **model_kwargs))
-                else:
+                if self.enable_streaming == "local":
```

Comment:
1. We should find a better name.
2. Why huggingface is better? In which case user should use "local", should we drop "local"?